internal/tekton-bundle/tekton-bundle_test.go

```diff
@@ -32,12 +32,12 @@ var _ = Describe("Tekton bundle", func() {
 
 	It("should return correct task path on okd", func() {
 		path := getTasksBundlePath(true)
-		Expect(path).To(Equal("/data/tekton-tasks/okd/kubevirt-tekton-tasks-okd-" + common.TektonTasksVersion + ".yaml"))
+		Expect(path).To(Equal("/data/tekton-tasks/okd/kubevirt-tekton-tasks-okd.yaml"))
```

Comment:
Can you please also remove `"kubevirt.io/ssp-operator/internal/common"` import, if it's not used?